src/Mantasflowers.Domain/Enums/OrderStatus.cs

```diff
@@ -0,0 +1,13 @@
+namespace Mantasflowers.Domain.Enums
+{
+    public enum OrderStatus
+    {
+        DRAFT,
```

Comment:
Consider keeping such state in client-side storage.

Reply:
Might be the case. But then we need some sort of "back end for front end". I.e. javascript based logic layer between UI and C# backend.

In the general case you are correct, not sure what Martynas wants to do though.